Src/DependencyCollector/NetCore/DependencyCollector.NetCore.csproj

```diff
@@ -1,10 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Copyright>Copyright � Microsoft. All Rights Reserved.</Copyright>
+    <Copyright>Copyright © Microsoft. All Rights Reserved.</Copyright>
     <VersionPrefix>2.4.0-beta1</VersionPrefix>
     <Authors>Microsoft</Authors>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFrameworks>netstandard1.3;netstandard1.5</TargetFrameworks>
```

Comment:
The AI SDK for .NET Core has been targeting 1.6 for the last five months or so.  Either we need to change it to "downgrade" to 1.3 or this attempt to support 1.3 is not relevant.

Now that you've got reflection code to get the assembly name from the runtime is there really still a need for two netstandard packages?